### DISCUSSION TITLE
Pages getting saved to Downloads, instead of prompting - on one of 2 PCs

### DISCUSSION THREAD

**Discussion by Andy2No** [gildas-lormeau/SingleFile#975] [Q&A] [answered]
I've had SingleFile working fine on a PC with Windows 7 Pro, for some time, but today I installed it on one with Linux (Pop!_OS 22.04, based on Ubuntu 22.04, with MATE added as the desktop environment after installation) - that was all freshly installed over the last couple of days.

I have the destination for downloads in FireFox set to prompt for a location ("Always ask you where to save files") on both machines, and the destination setting for SingleFile set to "save to file system" on both too.  That's working on both, for downloading files, or saving a web page normally (with File->Save as), but not when using "SingleFile->Save page with single file" on the Linux machine.  When I save a page with SingleFile on the Windows 7 PC, it works as expected, and prompts me for the location.  When I do it on the Linux machine, it just saves it straight to Downloads, without asking.

I've checked the version of SingleFile is the same one both - 1.21.11.

Am I missing a setting?

**Comment by gildas-lormeau** [comment, marked as answer]
You have to enable the option `File name > open the "Save as" dialog to confirm the file name` in SingleFile.

**Reply by Andy2No** [reply]
Thanks, @gildas-lormeau .  Yes, that worked.  I missed seeing that option.

I wonder if people would find it more easily under Destination, which is where I was looking.  That seems more logical to me, because it's part of where and how the page gets saved.  Just a thought.

**Comment by Crimsonlady22** [comment]
Andy great guy. Immature thought on my part. Pat  lovely , and lonely. Miss you aunt pat
